src/Qir/Runtime/stdlib/src/math.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#![deny(clippy::all, clippy::pedantic)]
+
+// TODO: transition math functions to `__quantum__rt` once compiler support is ready.
```

Comment:
Is there an issue for this TODO to help indicate when the comment can be successfully resolved?

Reply:
Added as https://github.com/microsoft/qsharp-compiler/issues/1557